generated/broker_1/broker_details.json

```diff
@@ -58,17 +58,17 @@
     {
       "employer_name": "National Network to End Domestic Abuse",
       "binder_payment_due": null,
-      "billing_report_date": "2017-04-23",
```

Comment:
just a suggestion, but: if before you start work, you do a simple regeneration (just changing the dates) and check it into master directly, and then do work on a branch, the PR contains only the relevant changes and not all the files which only changed dates.